pkg/networkmanager/interfaces.js

```diff
@@ -143,7 +143,7 @@ function NetworkManagerModel() {
     var self = this;
     var byteorder = null;
 
-    var client = cockpit.dbus("org.freedesktop.NetworkManager");
+    var client = cockpit.dbus("org.freedesktop.NetworkManager", { superuser: true });
```

Comment:
Yes, that's correct. And doubly so in cases where the service has this ability, but it's misconfiguration in a distro that is requiring root.